content/en/docs/doc-contributor-tools/linkchecker/README.md

```diff
@@ -0,0 +1,67 @@
+# Internal link checking tool
+
+You can use [htmltest](https://github.com/wjdp/htmltest) to check for broken links in [`/content/en/`](https://git.k8s.io/website/content/en/). This is useful when refactoring sections of content, moving pages around, or renaming files or page headers.
+
+## What it does and doesn't check
+
+The link checker scans generated HTML files, not raw markdown. You can see the full configuration in [`.htmltest.yml`](https://git.k8s.io/website/.htmltest.yml).
```

Comment:
Where are the generated HTML files located. Do contributors know where to look?
annoying nit: Markdown (be consistent on page)

The htmltest tool depends on a configuration file, [`.htmltest.yml`](...) to determine which content to examine.